account_lock_date_update/wizards/account_update_lock_date.xml

```diff
@@ -43,10 +43,6 @@
         <field name="name">Update accounting lock dates</field>
         <field name="parent_id" ref="account.menu_finance_entries_actions" />
         <field name="action" ref="account_update_lock_date_act_window" />
-        <field
-            name="groups_id"
-            eval="[(6, 0, [ref('account.group_account_manager')])]"
```

Comment:
Why are you removing this? AFAIK it's needed for not being showed to non managers.

Reply:
Isn't the menu item restricted automatically to members of this group now that the transient model of the action is restricted to this group?

Reply:
This came from the superseded PR: https://github.com/OCA/account-financial-tools/pull/1093/commits/d0e45e964fda2c14fea6c940898cd3b21b016e3c#diff-c3e08694fc0d14c655416e2deebff469b00172a70207a41b32ff291647194b8aL46-L49

As commented, I just pulled the changes and added my own, so as this one didn't receive any comments in particular, I kept it.

However, I can confirm @StefanRijnhart's comment: because of the group limit on the transient model, it only shows once you have the "Show full accounting features enabled" and are a Billing Administrator

Reply:
OK then